dataservice/api/study_file/schemas.py

```diff
@@ -3,9 +3,15 @@
 from dataservice.api.study_file.models import StudyFile
 from dataservice.api.common.schemas import BaseSchema, IndexdFileSchema
 from dataservice.extensions import ma
+from dataservice.api.genomic_file.schemas import AVAILABILITY_ENUM
```

Comment:
Maybe this enum is better define in `common.schemas.IndexdFileSchema`?

Reply:
i think this field belongs to our model

Reply:
But it's shared between all file types, isn't it?

Reply:
Yep, agreed. I think it makes more sense to put this in IndexFileSchema since both genomic files and study files use this enum

Reply:
yes both in study_file and genomic_file. yeah, you are right i can define in common schema